packages/cli/src/services/ownership.service.ts

```diff
@@ -0,0 +1,36 @@
+import Container, { Service } from 'typedi';
+import * as Db from '@/Db';
+import { CacheService } from './cache.service';
+import { RoleRepository } from '@/databases/repositories';
+import type { User } from '@/databases/entities/User';
+
+@Service()
+export class OwnershipService {
```

Comment:
every time we add a new service, it's a great opportunity to add unit tests for it while the service is still simple enough. 
can we add some unit tests please ?